screen-calendar-get.py

```diff
@@ -9,9 +9,9 @@
 from google.auth.transport.requests import Request
 from googleapiclient.discovery import build
 import outlook_util
-from utility import is_stale, update_svg
+from utility import is_stale, update_svg, configure_logging
 
-logging.root.setLevel(logging.INFO)
+configure_logging()
```

Comment:
Unified the logging into a uniform format with datetime, filename, line number.  This also reads the LogLevel from an environment variable, to help with troubleshooting.